lib/matplotlib/tests/test_ticker.py

```diff
@@ -443,6 +443,22 @@ def test_set_params(self):
         assert sym._subs == [2.0]
         assert sym.numticks == 8
 
+    def test_min_ticks(self):
+        """
+        Create 6 graphs and test if all of theses graphs have 3 ticks minimum
+        on the y axis. Values have been choosen in order to cover all the
+        cases. With only positive/negative value, both positive and negative
+        values, with close values an with distant values.
+        """
+        st = [-2.5, -2, -1, -0.01, 2, 2.5]
+        en = [-200, -2.005, 1, 0.01, 2.005, 200]
+        fig, axen = plt.subplots(2, 3)
```

Comment:
```suggestion
        starts = [-2.5, -2, -1, -0.01, 2, 2.5]
        ends = [-200, -2.005, 1, 0.01, 2.005, 200]
        fig, axs = plt.subplots(2, 3)
```

`starts` and `ends` is more readable, and `axs` is the canonical name we use for the Axes'es returned by `subplots()`.